lib/peep.rb

```diff
@@ -0,0 +1,42 @@
+require 'pg'
```

Comment:
I'm afraid I found the naming here a bit confusing. If this is the Peep class, what does the @peep variable represent?